tests/src/main/java/com/hazelcast/stabilizer/tests/map/MapHeapHogWarmupTest.java

```diff
@@ -0,0 +1,136 @@
+package com.hazelcast.stabilizer.tests.map;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IMap;
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.Logger;
+import com.hazelcast.stabilizer.test.TestContext;
+import com.hazelcast.stabilizer.test.annotations.Run;
+import com.hazelcast.stabilizer.test.annotations.Setup;
+import com.hazelcast.stabilizer.test.annotations.Verify;
+import com.hazelcast.stabilizer.test.annotations.Warmup;
+import com.hazelcast.stabilizer.test.utils.ThreadSpawner;
+
+import java.util.concurrent.TimeUnit;
+
+import static com.hazelcast.stabilizer.test.utils.TestUtils.humanReadableByteCount;
+import static com.hazelcast.stabilizer.tests.helpers.HazelcastTestUtils.isClient;
+import static com.hazelcast.stabilizer.tests.helpers.HazelcastTestUtils.nextKeyOwnedBy;
+import static com.hazelcast.stabilizer.utils.CommonUtils.sleepSeconds;
+import static java.lang.String.format;
+
+/**
+ * Fills up the cluster to a defined heap usage factor during warmup phase.
+ * <p/>
+ * This tests intentionally has an empty run phase.
+ */
+public class MapHeapHogWarmupTest {
+
+    private static final int MIN_CLUSTER_SIZE = 3;
```

Comment:
I can understand the intention. All operations are executed on local keys, so we don't want a changing cluster during the execution. So there is this wait check in the warmup.

I don't know if this is necessary though. Isn't the cluster already stable in the warmup phase? So maybe we can skip this whole part of waiting for that.